corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -864,7 +864,7 @@ def _configured_columns(self):
         """
         configured_columns = self.cleaned_data['columns']
         location = self.cleaned_data.get("location")
-        if location:
+        if location and location not in [c.get('property') for c in configured_columns]:
```

Comment:
nit: could avoid creating a list
```suggestion
        if location and all(location != c.get('property') for c in configured_columns):
```